spyderlib/tour.py

```diff
@@ -108,41 +118,72 @@ def get_tour(index):
+                           "programming workflow (like syntax highlighting and "
+                           "inline plots). You can read in depth about them "
+                           "<a href=\"%s\">here</a>.<br><br>"
+                           "It's also important to note that you can open as "
+                           "many consoles as you want, by going to the "
+                           "menu entry<br><br>"
+                           "<i>Consoles > Open an IPython console</i>.<br><br>"
+                           "Please click on the button below to run some simple "
+                           "code in this console. This will be useful to show "
+                           "you other important Spyder features." % \
+                           qtconsole_link),
+              'widgets': [sw.ipython_console],
+              'run': ["li = list(range(100))", "d = {'a': 1, 'b': 2}"]
               },
-
-             {'title': _("The Python console"),
-              'content': _("Now lets interact with the <b>IPython Console</b>."
-                           "<br><br><i>Decoration</i> included also."),
-              'widgets': [sw.console_external],
-              'interact': True},
-
+              
              {'title': _("The Variable Explorer"),
-              'content': _("Now lets interact with the <b>IPython Console</b>."
-                           "<br><br><i>Decoration</i> included also."),
+              'content': _("In this pane you can view and edit the variables "
+                           "generated during the execution of a program, or "
+                           "those entered directly in one of Spyder's "
+                           "consoles.<br><br>"
+                           "As you can see, the <b>Variable Explorer</b> shows "
+                           "the variables generated during the last step of "
+                           "this tour. By doing a mouse double-click on any "
+                           "of them, a new window will be opened, where you "
+                           "can inspect and modify its contents."),
               'widgets': [sw.variable_explorer],
               'interact': True},
 
+             {'title': _("The Python console"),
+              'content': _("You can also run your code on a <b>Python console</b>. "
+                           "These consoles are useful because they are created "
+                           "much faster than an IPython console, and let you "
+                           "run each file in a dedicated console<br><br>"
+                           "By pressing the button below, you will notice that "
+                           "<b>Python consoles</b> are also connected to the "
+                           "Variable Explorer, and that the last one only shows "
+                           "the variables of the currently focused console."),
+              'widgets': [sw.external_console, sw.variable_explorer],

```

Comment:
Ok, I'll change it. But my idea would be to use the new custom layouts functionality to always show this tour uisng the *Spyder* layout.

But that's for another PR :-)